src/libpriv/rpmostree-kernel.c

```diff
@@ -496,6 +498,18 @@ rpmostree_run_dracut (int     rootfs_dfd,
   g_auto(GLnxTmpfile) tmpf = { 0, };
   g_autoptr(GBytes) random_cpio_data = NULL;
 
+  gboolean renamed_etc = FALSE;
```

Comment:
Let's add a tiny comment here on why we want to ship `/etc/passwd` in the initrd?